content/en/security/code_security/software_composition_analysis/setup_static/_index.md

```diff
@@ -285,71 +285,141 @@ If you already have automatic scanning enabled for a repository, a manual upload
+<div class="alert alert-danger">
+Services defined with a **Software Catalog schema v2.x** do **not** support `codeLocations`. Existing definitions can be upgraded to the v3 schema in the [Software Catalog][24]. Once the migration is complete, changes may take up to 24 hours to apply to findings. If you are unable to upgrade to v3, Datadog will fall back to alternative linking techniques (described below). These rely on less precise heuristics, so accuracy may vary depending on the Code Security product and your use of other Datadog features.
+</div>
 
-For a repository match, if there is a service called `myservice` and
-the repository URL is `https://github.com/myorganization/myservice.git`, then,
-it associates `myservice` to all files in the repository.
+#### Example (v3 schema)
 
-If no repository match is found, Datadog attempts to find a match in the
-`path` of the file. If there is a service named `myservice`, and the path is `/path/to/myservice/foo.py`, the file is associated with `myservice` because the service name is part of the path. If two services are present
-in the path, the service name closest to the filename is selected.
-{{% /collapse-content %}}
+{{< code-block lang="yaml" filename="entity.datadog.yaml" collapsible="true" >}}
+apiVersion: v3
+kind: service
+metadata:
+  name: billing-service
+  owner: billing-team
+datadog:
+  codeLocations:
+    - repositoryURL: https://github.com/org/myrepo.git
+      paths:
+        - path/to/service/code/**
+    - repositoryURL: https://github.com/org/billing-service.git
+      paths:
+        - "**"
+{{< /code-block >}}
+
+#### SAST finding
+
+If a vulnerability appeared in `github.com/org/myrepo` at `/src/billing/models/payment.py`, then using the `codeLocations` for `billing-service` Datadog would add `billing-service` as an owning service. If your service defines an `owner` (see above), then Datadog will link that team to the finding too. In this case, the finding would be linked to the `billing-team`.
+
+#### SCA finding
+
+If a library was declared in `github.com/org/myrepo` at `/go.mod`, then Datadog would not match it to `billing-service`. 
+
+If it was instead declared in `github.com/org/billing-service` at `/go.mod`, then Datadog would match it to `billing-service` due to the `“**”` catch-all glob. Consequently, Datadog would link the finding to the `billing-team`.
+
+**Datadog will attempt to map a single finding to as many services as possible. If no matches are found, Datadog will continue onto our next linking method.**
+
+### When the Software Catalog cannot determine the service
+
+If the Software Catalog does not provide a match—either because the finding’s file path does not match any `codeLocations`, or because the service uses the v2.x schema—Datadog evaluates whether **Error Tracking** can identify the service associated with the code. Datadog uses only the last 30 days of Error Tracking data due to product [data-retention limits][25].
```

Comment:
```suggestion
If the Software Catalog does not provide a match, because the finding’s file path does not match any `codeLocations`, or because the service uses the v2.x schema, Datadog evaluates whether Error Tracking can identify the service associated with the code. Datadog uses only the last 30 days of Error Tracking data due to product [data-retention limits][25].
```